src/dsmr/parser.h

```diff
@@ -83,7 +87,11 @@ struct ParsedData<> {
 // Do not use F() for multiply-used strings (including strings used from
 // multiple template instantiations), that would result in multiple
 // instances of the string in the binary
+#ifndef ARDUINO_ARCH_ESP8266
 static constexpr char DUPLICATE_FIELD[] PROGMEM = "Duplicate field";
+#else
+static constexpr char DUPLICATE_FIELD[] = "Duplicate field";
```

Comment:
Issue #2072 was opened here: https://github.com/esp8266/Arduino/issues/2072

With regards to PROGMEM the discussion linked to below seems to make sense,
but I cannot say I really understand it.

http://stackoverflow.com/questions/35091862/inline-static-data-causes-a-section-type-conflict

2016-05-30 22:39 GMT+02:00 Matthijs Kooijman <notifications@github.com>:

> In src/dsmr/parser.h
> <https://github.com/matthijskooijman/arduino-dsmr/pull/3#discussion_r65104895>
> :
>
> >  static constexpr char DUPLICATE_FIELD[] PROGMEM = "Duplicate field";
> > +#else
> > +static constexpr char DUPLICATE_FIELD[] = "Duplicate field";
>
> Reported a bug with the ESP core yet? Any idea what PROGMEM is defined to
> to cause this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/matthijskooijman/arduino-dsmr/pull/3/files/27e2ace07f866f0285c58f57b6690c9dfc280bab#r65104895>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB7KB-rLKWFBDDTY6BFP2q7trvl3ujbJks5qG0sRgaJpZM4Im8LH>
> .
>